### DISCUSSION TITLE
Can I create a console application using .NET 6 on macOS?

### DISCUSSION THREAD

**Discussion by alataffective** [dotnet/maui#8956] [Q&A]
I'm trying to create a .NET 6 console application using Visual Studio for Mac 17.0.6 (build 4). However I get the error:
`/usr/local/share/dotnet/sdk/5.0.408/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(5,5): Error NETSDK1045: The current .NET SDK does not support targeting .NET 6.0.  Either target .NET 5.0 or lower, or use a version of the .NET SDK that supports .NET 6.0. (NETSDK1045)`. I have SDK 6.0.302 installed.

**Comment by xtuzy** [comment]
something break😂
